internal/deviations/deviations.go

```diff
@@ -11,12 +11,16 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
-// Package deviations documents and controls vendor deviation
-// workarounds that exist in the featureprofiles test suite.
 //
-// This is a temporary workaround and will be deprecated in the
-// future.
+// Package deviations documents defines arguments to enable
+// workarounds for a featureprofiles test suite.
+//
+// Deviations are useful to temporarily enable passing for other dependent
+// subcases to give visibility to incremental progress to achieving
+// compliance to test.
```

Comment:
can we include here:

1. what is the process to add a deviation -- is each one that is added created as a field in this file, with an associated github issue that notes why it exists?
2. what is the process to remove a deviation? i.e., we do not want these to be long-lived things, so there should be some process by which they are removed.
3. if i want to run the /actual/ tests, can we document how this is done -- this should be (imho) the default thing that is being run -- with special flags being needed to involve deviations (ideally, each deviation individually) such that different target types are supported.

Reply:
Good idea to clarify.  I proposed some text for you to review